docs/how-to/python_api.md

```diff
@@ -0,0 +1,188 @@
+## Use the Spark Client from Python 
+
+In the following, we show how to use the spark client to manage service accounts
+using Python.
+
+### Bind to Kubernetes
+
+The following snipped allows you to import relevant environment variables
+into a confined object, among which there should an auto-inference of your 
+kubeconfig file location. 
+
+```python
+import os
+from spark_client.domain import Defaults
+from spark_client.services import KubeInterface
+
+# Defaults for spark-client
+defaults = Defaults(dict(os.environ))  # General defaults
+
+# Create a interface connection to k8s
+kube_interface = KubeInterface(defaults.kube_config)
+```
+
+Notat that if you want to override some of these settings,
+you can extend the `Default` class accordingly. 
+
+Alternatively you can also use auto-inference using the `kubectl` command via
```

Comment:
Should this be available with LightKube as well?

Reply:
Once the LightKube stuff is merged, we can probably update this section accordingly or put a note that both functionalities can be obtained using LightKube implementation, with no dependency on kubectl